extensions/github/src/components/DiscussionListItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { List, Icon, Color, ActionPanel, Action } from "@raycast/api";
+import { format } from "date-fns";
+
+import { DiscussionFieldsFragment } from "../generated/graphql";
+import { getGitHubUser } from "../helpers/users";
+
+function getDiscussionIcon(discussion: DiscussionFieldsFragment): List.Item.Props["icon"] {
+  const categoryText = discussion.category?.name ? `Category: ${discussion.category?.name}` : "Unknown";
+  const emojiHTML = discussion.category?.emojiHTML as string | null | undefined;
+  if (!emojiHTML) {
+    return { value: { source: discussion.repository?.owner?.avatarUrl }, tooltip: categoryText };
+  }
+  const fallbackImage = () => {
+    // note: getting the emoji directly break raycast. Looks like encoding issues.
+    const re = /fallback-src="(.*)"/;
+    const match = emojiHTML.match(re);
+    if (match) {
+      return match[1];
+    }
+    return undefined;
+  };
+  const fb = fallbackImage();
+  if (fb) {
+    return {
+      value: { source: fb },
+      tooltip: categoryText,
+    };
+  }
+  return fb;
+}
+
+export function DiscussionListItem(props: { discussion: DiscussionFieldsFragment }): JSX.Element {
+  const d = props.discussion;
+  const user = getGitHubUser(d.author);
+  return (
+    <List.Item
+      icon={getDiscussionIcon(d)}
+      title={d.title}
+      subtitle={d.repository?.nameWithOwner}
+      accessories={[
+        {
+          icon: { source: d.answer ? Icon.Checkmark : "", tintColor: Color.Green },
```

Comment:
```suggestion
          icon: { source: d.answer ? Icon.Checkmark : undefined, tintColor: Color.Green },
```

Reply:
I got an error when I do that

<img width="789" alt="CleanShot 2022-11-21 at 13 10 31@2x" src="https://user-images.githubusercontent.com/3163807/203050652-575b6c1f-7cff-4380-bd87-7531470f3550.png">

Reply:
ah right sorry, should be this:
```suggestion
          icon: d.answer ? { source: Icon.Checkmark, tintColor: Color.Green } : undefined,
```